### ISSUE TITLE
Sidebar visual glitch

### ISSUE THREAD

**Issue by rayman89** [tim-we/tabs-aside#35]
When you get over 99 tabs the numbers display badly. Increasing the width of the sidebar doesnt fix it. I'm on firefox 62beta on win 7 x64

![image](https://user-images.githubusercontent.com/5449042/42176718-8e3b311c-7def-11e8-91a3-c621beb577e2.png)

**Comment by tim-we** [comment]
I will try to fix this with version 3. Never thought someone would have that many tabs open.
What are your font settings? I could not reproduce this with 100+ tabs. I

**Comment by rayman89** [comment]
That's odd I thought the addon was clearly designed for people that has way too many tabs open. I'm always around 300+ open and with 2,5k+ bookmarks.

I did not change the default font. Maybe it has to do with the screen resolution? I'm using 1824*1026

**Comment by tim-we** [comment]
Yes it is designed for people with too many tabs open. For me and people I know that was about 20-40 tabs. But you are not the first one with that many tabs, I implemented the lazy tab loading because some guy crashed his Firefox trying to restore 100+ tabs.
